dkg/messages.go

```diff
@@ -150,6 +158,55 @@ func (msg *Init) Decode(data []byte) error {
 	return json.Unmarshal(data, msg)
 }
 
+// Reshare triggers the resharing protocol
+type Reshare struct {
+	// ValidatorPK is the the public key to be reshared
+	ValidatorPK types.ValidatorPK
+	// EncryptedShares contains the encrypted shares of the old set
+	EncryptedShares map[types.OperatorID][]byte
+	// ThresholdOld is the threshold of the old set
+	ThresholdOld uint16
```

Comment:
should already hav that info

Reply:
The node can fetch the info and prepare this struct and pass it to the protocol. Otherwise the protocol need to fetch directly (from storage)?

Reply:
Removed  these two fields and instead node will retrieve the `KeyGenOutput` from local storage.  https://github.com/bloxapp/ssv-spec/pull/82/commits/dbce44e474475b6485d877e6f7f2a9de70b11c18